drake/systems/plants/@Manipulator/Manipulator.m

```diff
@@ -92,6 +92,68 @@
 
     end
 
+    function [xdot,dxdot,ps,qs,qds,us,ts] = dynamicsSym(obj,x,u,t)
```

Comment:
i really don't like this function.  it should not live in manipulator.  I would expect an external method which calls dynamics with symbolic inputs.  We already have those, for instance in generateGradients.  
also, the inputs to all dynamics methods are t,x,u (not x,u,t)